modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,197 @@
+
+[NOTE]
+=====
+Data Mover supports backup and restore of CSI volume snapshots only.
+=====
+
+.Prerequisites
+
+* Have a stateful application running in a separate namespace.
+
+* The `StorageClass` and `VolumeShapshotClass` CRs support CSI.
+
+* Only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
+
+* Only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
+
+* Include the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR to indicate for Velero which to use.
+
+* Install the VolSync Operator using the Operator Lifecycle Manager (OLM). Administrators can link:https://docs.openshift.com/container-platform/4.10/operators/admin/olm-adding-operators-to-cluster.html[install Operators on clusters]. Non-administrators can link:https://docs.openshift.com/container-platform/4.10/operators/user/olm-installing-operators-in-namespace.html[install Operators in namespaces].
++
+[NOTE]
+====
+The VolSync Operator is required only for use with the Technology Preview Data Mover. The Operator is not required for using OADP GA features.
+====
+
+* Install the link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[OADP Operator] using OLM.
+
+* Install the link:https://volsync.readthedocs.io/en/stable/installation/index.html[VolSync operator] using OLM.
+
+.Procedure
+
+. Configure a link:https://volsync.readthedocs.io/en/stable/usage/restic/index.html#id2[Restic secret].
++
+[source,yaml]
+----
+$ cat << EOF > ./restic-secret.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret-name>
+type: Opaque
+stringData:
+  # The repository encryption key
+  RESTIC_PASSWORD: <secure-restic-password>
+EOF
+----
+
+. Create a Data Protection Application (DPA) similar to the following example. Note that `defaultPlugins` includes CSI.
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  features:
+    enableDataMover: true
+    dataMoverCredential: <secret-name>  <1>
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <bucket-name>
+          prefix: <bucket-prefix>
+        provider: aws
+  configuration:
+    restic:
+      enable: <true or false>
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - csi
+      featureFlags:
+        - EnableCSI
+----
+<1> Add the Restic secret name from the previous step. If this is not done,  the default secret name `dm-credential` is used.
++
+The OADP operator installs two VolumeSnapshotMover custom resource definitions (CRDs), `VolumeSnapshotBackup` and `VolumeSnapshotRestore`.
```

Comment:
no need for `VolumeSnapshotMover` part, as there is not such kind and it might create a confusion. 
i.e:
The OADP operator installs two Custom Resource Definitions (CRDs): `VolumeSnapshotBackup` and `VolumeSnapshotRestore`.

Reply:
Done